### PR TITLE
Add Linux cheatsheets

### PR DESCRIPTION
**Pull request by martialblog**
This PR adds Linux Cheatsheets that can be viewed online or be downloaded on training VMs.

Adding them to this repository has these benefits: 

* already established repo that is included in most trainings
* public repo that can be viewed by everyone, markdown is rendered as HTML
* public repo that can simply be downloaded in training VM provisioning

The cheatsheets are a bare minimum from my point of view. Can be extended in the future

### CODE REVIEW

**Comment by martialblog** [comment]
@dgoetz @mwaldmueller @mocdaniel @theFeu @widhalmt @RincewindsHat 

Feedback is welcome

File: extras/Linux-Cheatsheet-English.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+| Description | Command |
+|---|---|
+| Clear terminal | `clear` |
+| Show path of the current directory | `pwd` |
+| Change into a directory | `cd <Path to directory>` |
+| List a directory | `ls -l <Path to directory>` |
+| Create directory | `mkdir <Path to directory>` |
+| Remove empty directory | `rmdir <Path to directory>` |
+| Remove directory | `rm -rf <Path to directory>` |
```
Comment (mkayontour) [inline extras/Linux-Cheatsheet-English.md]: Remove directory with force is maybe not the best option for unexperienced users.

`rm -ri` for interactive mode could be a solution.

Comment (martialblog) [reply extras/Linux-Cheatsheet-English.md]: Agreed. Fixed it.

**Comment by mocdaniel** [comment]
LGTM, the selection of mentioned commands fits the purpose, too. I'd like to see some kind of 'sectioning' so participants will be able to find what they're looking for at first glance.

Suggestion:

- Navigation on the CLI
- Managing files and directories
- Viewing and manipulating file contents
- Managing software on your system

**Comment by martialblog** [comment]
@mocdaniel makes sense, especially once the list grows

**Comment by dgoetz** [comment]
Some small things to add:
- A section with typically paths and the difference between relative and absolute path
- Permissions and Ownership including chown/chmod (chmod best with letters and not digits, to avoid a common pitfall)
- Copy and move can handle multiple sources when destination is a directory

**Comment by martialblog** [comment]
Done. Thanks Dirk!

**Pull request merged**